test/event_handler_test.py

```diff
@@ -238,6 +238,10 @@ def test_records_error_but_does_delete_messages_for_duplicate_events(self):
                 ('event-recorder', 'INFO', 'Decrypted event with ID: sample-id-1'),
                 ('event-recorder', 'WARNING', 'Failed to store an audit event. The Event ID sample-id-1 already exists in the database'),
                 ('event-recorder', 'INFO', 'Stored audit event: sample-id-1'),
+                ('event-recorder',
+                 'WARNING',
+                 'Failed to store a billing event. The Event ID sample-id-1 already exists in '
+                 'the database'),
```

Comment:
If we're chopping down the arguments, does the string _also_ need to be split? The line isn't _that_ long... 😅

Reply:
Ah, I think the IDE did that 🙄 I'll fix it.